core/log/log_pb/log.proto

```diff
@@ -46,13 +46,24 @@ message Message {
 
     // The key-value pairs of extra data.
     repeated Value values = 8;
+
+    // The backtraces passed along with the log message (e.g. a Java exception).
+    repeated Backtrace backtrace = 9;
```

Comment:
I've been considering adding this for go chained errors too. We already call this "Cause", and it would be nice to keep this name for consistency's sake.